File: bronze/2547/c++/main.cpp
#include <iostream>

int main()
{

    int numit, num;
    long long temp;
    __int128 sum = 0;
    std::cin >> numit;

    for (int i = 0; i < numit; i++) {
        sum = 0;
        std::cin >> num;
        for (int _ = 0; _ < num; _++) {
            std::cin >> temp;
            sum += temp;
        }
        std::cout << ((sum % num == 0) ? "YES" : "NO") << '\n';
    }

    return 0;
}

File: bronze/2547/python/main.py

if (__name__ == "__main__"):

    numit = int(input())
    count = 0
    while (count < numit):
        try:
            sum = 0
            num = int(input())
            for _ in range(num):
                temp = int(input())
                sum += temp
            if (sum % num == 0): print("YES")
            else: print("NO")
            count += 1
        except: 
            continue